docs/layers.md

```diff
@@ -740,7 +740,9 @@ Places with `kind` values of `continent`, `country`, with others added starting
 #### Place properties (common optional):
 
 * `country_capital`: a `true` value normalizes values between OpenStreetMap and Natural Earth for kinds of `Admin-0 capital`, `Admin-0 capital alt`, and `Admin-0 region capital`.
+* `country_capital:xx`: when present, either `true` or `false` to override the `country_capital` value for XX's viewpoint.
```

Comment:
[docs] We should reference it uses the same points-of-view as `kind:xx`.

Reply:
Good idea! Fixed in 13257b22.